rdr_service/offline/enrollment_check.py

```diff
@@ -0,0 +1,95 @@
+from datetime import datetime, timedelta
+
+from rdr_service.dao.participant_summary_dao import ParticipantSummaryDao
+from rdr_service.model.participant_summary import ParticipantSummary
+from rdr_service.participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, SampleStatus
+from rdr_service.services.jira_utils import JiraTicketHandler
+
+PS_DAO = ParticipantSummaryDao()
+THIRTYONE_DAYS = datetime.now() - timedelta(days=31)
+
+
+def check_enrollment(create_ticket=True):
+    """For members who are listed as full-member,
+     check that necessary requirements are valid"""
+    members = get_last_30_days()
+    results = None
+    if members:
+        results = check_requirements(members)
+    if results:
+        # make a roc ticket
+        if create_ticket:
+            jira_handler = JiraTicketHandler()
+            jira_handler.create_ticket('Participant Status offline job', str(results).strip('[]'), issue_type='Task',
+                                       board_id='ROC')
+        return False
+    else:
+        return True
+
+
+def get_last_30_days():
```

Comment:
Why just 30 days?

Reply:
it runs once a month. The actual number is 31 days. Looks like I forgot to change the name of the function.

Reply:
renamed